src/ServiceProvider/AuthFlowServiceInterface.php

```diff
@@ -32,9 +32,10 @@ public function getAuthenticatedUri(ServerRequestEx $request) : XUri;
 
     /**
      * @param XUri $returnUri
+     * @param bool $sha512EncryptionEnabled
```

Comment:
might want to update the docblock to match the function signature

Reply:
we might end up changing this a bit. we need support multiple versions Sha256 since some IdPs dont always support the 512 version.